enable LMS insecure connections (#18608)
cockroachdb/docs: src/current/v23.2/live-migration-service.md

`lms.replicaCount` determines the number of LMS instances created as `lms` pods on the Kubernetes cluster, across which application traffic is distributed. This defaults to `3`.

#### LMS connection

~~~ yaml
lms:
  allowTLSDisable: false
~~~

> `lms.allowTLSDisable` specifies whether insecure LMS connections to databases are allowed. This should be set to `true` **only** if secure SSL/TLS connections to the source or target database are not possible. [Secure SSL/TLS connections](#security) should be used by default.

Reviewer: ```suggestion
`lms.allowTLSDisable` enables insecure LMS connections to databases, and is disabled by default. Enable insecure connections **only** if a secure SSL/TLS connections to the source or target database is not possible. When possible, [secure SSL/TLS connections](#security) should be used.
```
Completely optional, just trying out streamlining this a bit.

{{site.data.alerts.callout_success}}

> You can also set the `--allow-tls-mode-disable` [flag](#global-flags) to enable insecure LMS connections.

Reviewer: ```suggestion
 The `--allow-tls-mode-disable` [flag](#global-flags) enables insecure LMS connections by setting `lms.allowTLSDisable` to `true`.
```
Is this how it works?
